src/Table/write_fits.cxx

```diff
@@ -163,6 +163,81 @@ void write_element_given_column_and_row(fitsfile *fits_file, int fits_type,
 /**********************************************************/
 /**********************************************************/
 
+// Labeled_properties are stored as keywords whose values are strings
+// of the form
+
+// label + DOT + XMLATTR_DOT + ATTR_IRSA_VALUE : value  (for prop.value_)
+// label + DOT + XMLATTR_DOT + attrname : attrvalue  (for elements of
+// prop.attributes_)
+
+// where label includes value of prop's ATTR_NAME attribute for
+// uniqueness.
```

Comment:
We call 'fits_write_key_longstr()' below rather than 'fits_write_key_str()'; does that address your concern.

Please note that I am not trying to improve functionality (or fix bugs) in this PR.  All I've done is pull out code from write_fits() into a separate function.

Reply:
I pulled out the corresponding code into a separate function on the debian11 branch.  I'm hoping that doing the same thing in master will simplify future merges.

Reply:
I mean non-standard keyword in a fits header written by this package such as:
     "HIERARCH VOTABLE.RESOURCE.INFO.x1.<xmlattr>.name" 
may violate the FITS standard, unless I missed some rules. Usually it is 1-8 bytes.

Reply:
The debian11 branch addresses this issue by putting all of what would ordinarily be the keyword-name info into the keyword-value and using more or less sequential numeric identifiers as keyword-names.  There was some reason why I couldn't just backport the entire debian11 solution (which has its own problems) to debian8, but yes, there is a lot of work to be done on FITS support generally and on FITS support for keywords in particular.  All I'm hoping to do in this PR, though, is (1) pull out existing code into a separate function, and (2) not add any new bugs.